src/components/FormPluginEditor/M3RichTextEditor.module.scss

```
@@ -112,6 +112,89 @@ $m3-state-active: rgba(39, 50, 112, 0.12);
     }
 }
 
+// Fluid card (#594): hosts that are not the settings deck — the public
+// tenant-onboarding DPA step and the DPA blocker — give the card their own
+// column width. The desktop rule above pins the deck card to 800x740, which
+// would overflow a sign-in-sized column.
+//
```

Stylelint (17.14.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)